src/components/HelpMark/__stories__/HelpMark.stories.tsx

```diff
@@ -28,4 +35,29 @@ export default {
 
 type Story = StoryObj<typeof HelpMark>;
 
-export const Default: Story = {args: {children: 'Some content'}};
+export const Default: Story = {};
+
+export const Size: Story = {
```

Comment:
Better use mdx, and include this different variations via `<Canvas/>` component, this will utilize default Storybook behavior with story code switcher

For example, i changed Select stories this way

https://github.com/gravity-ui/uikit/blob/e2b91bf0c6e20f9e2ca7a14539342cc69885ad07/src/components/Select/__stories__/Showcase.mdx?plain=1#L4-L6